.github/workflows/lint-test.yaml

```diff
@@ -6,7 +6,7 @@ jobs:
   linter-artifacthub:
     runs-on: ubuntu-20.04
     container:
-      image: artifacthub/ah
+      image: artifacthub/ah:v1.2.0
```

Comment:
@dagl you can check on https://github.com/artifacthub/hub/blob/c75f1e715fead884840a3ab7fc5441c328c61c61/.github/workflows/ci.yml#L28-L38 how the new version of the `ah` lint command can be configured on the CI to move forward instead of staying stuck with the old version